server/database/default.json

```diff
@@ -0,0 +1,3 @@
+{
+  "ConnectionUrl": "mongodb+srv://greg34910:cookie123@cluster0-pf0b2.mongodb.net/test?retryWrites=true&w=majority"
```

Comment:
This is using your own personal DB connection. Can you make this generic?

Reply:
Hidden